portfolio/src/main/webapp/index.html

```diff
@@ -15,6 +15,14 @@
         <script src="business-logic/FilterController.js"></script>
         <script src="business-logic/Test.js"></script>
         <!-- End of Business Logic -->
+
+        <!-- Comments Section Logic -->
+        <script src="comments/CommentFetcher.js"></script>
+        <script src="comments/CommentRenderer.js"></script>
+        <script src="comments/CommentFacade.js"></script>
+        <script src="comments/comments-setup.js"></script>
```

Comment:
Why is this one hyphenated?

Reply:
Because it's just a script and not a class.